### PR TITLE
[BPK-1243] expose radii on Android

### PR DESCRIPTION
**Pull request by matteo-hertel**
I've also added a coming soon message to the main app as part of the PR

screenshot_1518101001

### CODE REVIEW

File: .idea/modules.xml [hunk 1]
```diff
@@ -3,11 +3,8 @@
   <component name="ProjectModuleManager">
     <modules>
       <module fileurl="file://$PROJECT_DIR$/Backpack/Backpack.iml" filepath="$PROJECT_DIR$/Backpack/Backpack.iml" />
-      <module fileurl="file://$PROJECT_DIR$/Backpack/Backpack.iml" filepath="$PROJECT_DIR$/Backpack/Backpack.iml" />
```
Comment (matthewdavidson) [inline .idea/modules.xml]: this file should be excluded from source control, right?

Comment (matteo-hertel) [reply .idea/modules.xml]: There is a default gitignore when creating and android app, I think we're using that one and it's blacklisting a bunch of files from the `.idea` folder but not all of it
https://github.com/Skyscanner/backpack-android/blob/master/.gitignore#L39

Comment (matthewdavidson) [reply .idea/modules.xml]: Ah that would imply that some of them should be in source control ... 🤔

Comment (matteo-hertel) [reply .idea/modules.xml]: THat would be me guess but I might be completely wrong

Comment (matteo-hertel) [reply .idea/modules.xml]: Also this https://stackoverflow.com/questions/26336709/android-studio-should-the-entire-idea-directory-be-in-git-ignore

Comment (matthewdavidson) [reply .idea/modules.xml]: Wanna try removing it in a separate PR?

Comment (matteo-hertel) [reply .idea/modules.xml]: I'll check if our gitignore is different from a default android one and change it accordgly 👍

**Pull request merged**